wgpu-hal/src/gles/command.rs

```diff
@@ -28,6 +28,7 @@ pub(super) struct State {
     has_pass_label: bool,
     instance_vbuf_mask: usize,
     dirty_vbuf_mask: usize,
+    push_offset_to_uniform: Vec<super::UniformDesc>,
```

Comment:
looks like this is the only part that's heap allocated. This isn't a problem in itself, but we could use `ArrayVec` to solve it

Reply:
`ArrayVec` needs for `T` to be `Copy` when using `try_extend_from_slice` and `UniformDesc`doesn't implement it nor can it, since `glow::UniformLocation` isn't `Copy`

Reply:
`extend_from_slice` is just a convenience method, which btw isn't requiring `Copy` in std. So let's just `extend()` instead.